src/components/Profile/components/PersonalInfoList/UpdateUserPrivacyDialog/index.jsx

```diff
@@ -0,0 +1,80 @@
+import { FormControl, IconButton, Input, InputLabel } from '@mui/material';
+import GordonSnackbar from 'components/Snackbar';
+import { useState, useEffect } from 'react';
+import userService from 'services/user';
+import SearchField from 'views/PeopleSearch/components/SearchFieldList/components/SearchField';
+
+const UpdateUserPrivacy = (username, field) => {
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+  const [group, setGroup] = useState('');
+  const [groupList, setGroupList] = useState([]);
+  let visibleTo = '';
+
+  const handlePrivacy = async (event) => {
+    try {
+      await userService.setUserPrivacy({ Field: field, VisibilityGroup: event.target.value });
+      setSnackbar({
+        message: 'Your office location will update within a couple hours.',
+        severity: 'success',
+        open: true,
+      });
+    } catch {
+      setSnackbar({
+        message: 'Office location failed to update. Please contact CTS.',
+        severity: 'error',
+        open: true,
+      });
+    }
+  };
+
+  useEffect(() => {
+    userService.getVisibilityGroups().then(setGroupList);
+  }, []);
+
+  useEffect(() => {
+    userService.getPrivacySetting(username).then(setGroup);
+  }, [group]);
+
+  let tempField = field;
+  if (field == 'HomeCity HomeState') {
+    field = 'HomeCity';
+  } else if (field == 'Country HomeCountry') {
+    field = 'Country';
+  }
+
+  // get user's privacy setting (Public, Privacy, FacStaff) for this field
+  for (let i = 0; i < group.length; i++) {
+    if (group[i].Field === field) {
+      visibleTo = group[i].VisibilityGroup;
+    }
+  }
+
+  field = tempField;
+
+  if (field == 'HomeCity HomeState') {
+    visibleTo = group[i].VisibilityGroup;
+  }
```

Comment:
```suggestion
  if (field == 'HomeCity HomeState') {
    visibleTo = group[i].VisibilityGroup;
  }
```
why are we only doing this for HomeCity HomeState?

Reply:
This is because only the button in Home field need to control more than one field in the backend (HomeCity, HomeState, HomeCountry, Country)